docs/index.yaml

```diff
@@ -2,21 +2,52 @@ apiVersion: v1
 entries:
   illumidesk:
   - apiVersion: v2
-    appVersion: illumidesk
-    created: "2020-10-22T18:57:09.705749-04:00"
+    appVersion: 0.1.0
+    created: "2020-11-13T20:16:56.722797-05:00"
     dependencies:
     - import-values:
       - child: rbac
         parent: rbac
       name: jupyterhub
       repository: https://jupyterhub.github.io/helm-chart/
-      version: 0.9.1
+      version: 0.10.2
     - condition: postgresql.enabled
       name: postgresql
       repository: https://charts.bitnami.com/bitnami
-      version: 9.8.4
+      version: 9.8.6
+    - condition: datadog.enabled
+      name: datadog
+      repository: https://helm.datadoghq.com
+      version: 2.4.29
     description: An extention of the JupyterHub chart with additional IllumiDesk resources
-    digest: d224b37c673b07ec377095833fc63757879461fae41f7f013a2d2677e013f52f
+    digest: 0ff527bd5d47a14440c9ad0f6f93fb43670ca322d77780e0b97db89373679795
+    icon: https://configs.illumidesk.com/images/illumidesk-80.png
+    maintainers:
+    - email: hello@illumidesk.com
+      name: IllumiDesk Team
+    name: illumidesk
+    urls:
+    - illumidesk-0.1.0.tgz
+    version: 0.1.0
+  - apiVersion: v2
+    created: "2020-11-13T20:16:56.706441-05:00"
+    dependencies:
+    - import-values:
+      - child: rbac
+        parent: rbac
+      name: jupyterhub
+      repository: https://jupyterhub.github.io/helm-chart/
+      version: 0.10.2
```

Comment:
Is this correct? It looks like are the same for both versions (0.1.0 and 0.0.1), I believe the jupyterhub upgrade was recent after the last release right?